crates/extensions/tedge_health_ext/src/tests.rs

```diff
@@ -0,0 +1,59 @@
+use std::time::Duration;
+
+use crate::HealthMonitorBuilder;
+use crate::HealthMonitorMessageBox;
+use tedge_actors::Builder;
+use tedge_actors::ReceiveMessages;
+use tedge_actors::Runtime;
+use tedge_actors::ServiceConsumer;
+use tedge_actors::SimpleMessageBoxBuilder;
+use tedge_mqtt_ext::MqttMessage;
+use tedge_mqtt_ext::Topic;
+use tokio::time::timeout;
+const TEST_TIMEOUT: Duration = Duration::from_secs(10);
+
+#[tokio::test]
+async fn send_health_check_message_to_generic_topic() -> Result<(), anyhow::Error> {
+    let mut mqtt_message_box = spawn_a_health_check_actor("health-check-service-1").await?;
+    let health_check_request = MqttMessage::new(&Topic::new_unchecked("tedge/health-check"), "");
+    mqtt_message_box.send(health_check_request).await.unwrap();
+
+    if let Some(message) = timeout(TEST_TIMEOUT, mqtt_message_box.recv()).await? {
+        dbg!(message.payload_str().unwrap());
+        assert!(message.payload_str()?.contains("up"))
```

Comment:
This check could be a bit more strict. What about the topic for instance?

* [minor] please remove the `dbg!`